scanner/matcher/matcher.go

```diff
@@ -114,12 +118,28 @@ func NewMatcher(ctx context.Context, cfg config.MatcherConfig) (Matcher, error)
 		}
 	}()
 
+	// Note: http.DefaultTransport has already been modified to handle configured proxies.
+	// See scanner/cmd/scanner/main.go.
+	defaultTransport := http.DefaultTransport
+	// If this is a release build, Matcher should only reach out to Central,
+	// so deny (and log) any other traffic.
+	if buildinfo.ReleaseBuild {
+		defaultTransport = httputil.DenyTransport
+	}
+	// Matcher should never reach out to Sensor, so ensure all Sensor-traffic is denied.
+	transport, err := httputil.TransportMux(defaultTransport, httputil.WithDenySensor(true))
```

Comment:
super nit: change variable name to something other than `defaultTransport` as it may not represent the default transport (perhaps `baseTransport`?)

Reply:
`defaultTransport` is definitely not `http.DefaultTransport` as you say, but I think `baseTransport` isn't quite right. `baseTransport` makes me think it's the `http.RoundTripper` that will do the heavy lifting underneath some wrapper which does some added work. I think `defaultTransport` makes sense, as it's the one to use by default when not communicating with Rox-services